### DISCUSSION TITLE
R-Link 2 24 pin diagram

### DISCUSSION THREAD

**Discussion by larskka** [cedricp/ddt4all#1040] [Q&A]
Hy hope someone can help me. have a scenic iv from 2020 where i want to install reverse camera. anyone can help with a diagram to r-link 2 with 24 pin have tryed 18 cam+ 19 cam - 20 cam + suply but i dont work. thr camera an wire work when i connect to another schreen, so i think it is the wiring there is wrong...

**Comment by larskka** [comment]
![stik](https://github.com/cedricp/ddt4all/assets/161332104/0c78d5c8-09b4-4bd5-87ac-f5c7f8c3f26a)
this is what i have tried drawn as looking inside the connection on r-link

**Comment by AndyIreland** [comment]
if its a car without a reverse camera and you have put it on as an add on/upgrade I should imagine you will have to enable rear reverse camera in the navigation section with DDT4ALL on your rlink2

**Reply by youceffeddal84** [reply]
rlink2 scenic 4 location can someone help me how to install a new android multimedia head unit as I couldn't the location of rlink2 module

**Reply by youceffeddal84** [reply]
where is rlink2 scenic 4 location can someone help me how to install a new android multimedia head unit as I couldn't find the location of rlink2 module.if it is begind the glove box can anyone show me how to access to it

**Comment by larskka** [comment]
Yes its is new and activated with ddt4all. it has now the camera menu, but says defet camera or missing signal. so i think i dident use the right pins? i have bought at adapter  but it is 32 pin and the radio has 24 pin so i found the connections on the picture but i dont work

**Comment by AndyIreland** [comment]
is your camera a 12v or 6v power requirement?

**Comment by larskka** [comment]
it is 6 volt. if i connect camera wit power from extern cable there is still no video signal on r link . but there is picture if i connect the video cable to a old tv...

**Comment by AndyIreland** [comment]
strange , should work then you would think. - so you are using the +6v and -6v pins and the CVBs  + and - and putting car into reverse gear and no pic ? - where did you get the 24pin wiring detail diagram from. 

here is wiring for rlink1 but I dont know if it the same for rlink2:
![image](https://github.com/cedricp/ddt4all/assets/49172217/214e8b74-beb3-4e97-bc61-9adf413319c7)

![image](https://github.com/cedricp/ddt4all/assets/49172217/92491077-3800-42ca-bd8d-3bb26d8ad80b)
![image](https://github.com/cedricp/ddt4all/assets/49172217/a87ce102-a2d7-4d8e-8203-8ed32bb5efa2)
![image](https://github.com/cedricp/ddt4all/assets/49172217/8e5bc79c-f16d-4c5a-b0e2-d7b484e9cf82)

**Comment by larskka** [comment]
I think I saw it on this site and it fit fine with the pins that were not in use. so just like you have shown here but - for the camera supply I have taken out the rear lights. But it is not only the power that has problems, because I have tried to supply the camera with external power, so the video signal is also wrong. When the car is put into reverse gear, a black screen appears and it reports a camera error or lack of signal

**Comment by AndyIreland** [comment]
yes, it could possibly be different wiring for the rlink2 vs  the rlink1 24 pin connector then

**Comment by larskka** [comment]
My thought is that mabye there i a different wirring in r-link2 ?

**Comment by AndyIreland** [comment]
I found this - pin 21 could it be on the 32pin cable?
https://github.com/cedricp/ddt4all/issues/696

![image](https://github.com/cedricp/ddt4all/assets/49172217/232b9b6a-b2f1-4444-bec3-d67d474f43ee)

**Comment by larskka** [comment]
the emty wire holes in my radio is 1,3,4,5,12,14,16,18,19,20,21 . when i have been surfing on the web i have seen a lot of r-link 2 with 32 pin connector, the problem is that mine is a 24 pin and cant find any diagram with 24 pin for r-link 2 so i could be sure of which wires i shoul use

**Comment by larskka** [comment]
Thank you Andy, I hope someone have the wire connection for r-link 2 so i for sure can find the faul

**Comment by AndyIreland** [comment]
havent found a wiring diagram - but this sounds like the kind of lead you need maybe if yours is a 24pin plug:
https://www.newegg.ca/p/0JF-05RE-006E3

![image](https://github.com/cedricp/ddt4all/assets/49172217/32fa0b70-2327-48c1-92b1-f67c0e3be1ec)

**Comment by larskka** [comment]
Yes it would be great, but im not sure if the wirring is correct. Its for the elder medinav, i have seen som sellers write , that it wont support r-link?

**Comment by matzel82** [comment]
You need a diagram for the connection ? I can give you… I need your VIN

the lead at the picture is for MediaNav UlC4 not for RLink. It’s different.

**Comment by matzel82** [comment]
I think that’s important to config the camera at RLink. To „yes“

**Comment by larskka** [comment]
Vin nr.  vf1rfa00063460938 thankyou

**Comment by larskka** [comment]
> You need a diagram for the connection ? I can give you… I need your VIN
> 
> the lead at the picture is for MediaNav UlC4 not for RLink. It’s different.

vf1rfa00063460938 thank you :-)

**Comment by matzel82** [comment]
![IMG_2259](https://github.com/cedricp/ddt4all/assets/31157216/ac880433-731d-417a-b71f-1da958c354b0)
![IMG_2260](https://github.com/cedricp/ddt4all/assets/31157216/ed6c7295-c258-42c2-84af-89e73ce93702)
![IMG_2261](https://github.com/cedricp/ddt4all/assets/31157216/41005839-736a-4404-b62d-b39cf7474c24)
![IMG_2262](https://github.com/cedricp/ddt4all/assets/31157216/81d68dcb-c584-4dd0-af84-f24e4df68c89)
![IMG_2263](https://github.com/cedricp/ddt4all/assets/31157216/cdbd2139-37a4-4943-a869-58252d756f67)
![IMG_2264](https://github.com/cedricp/ddt4all/assets/31157216/fa4904d9-b275-475e-af64-a72a282589f7)

**Comment by matzel82** [comment]
Apropos… you have to visit a garage, there is a OTS to reprogramming your ECUs

**Comment by larskka** [comment]
My German is not the Best  ;-) is it corrett i should use pin 2,3,5.  The car already have a wire in pin 2?
The empty pin Are 1,3,4,5,12,14,16,18,19,20,21.

**Comment by matzel82** [comment]
Im not shure if you have the correct connection. Normally you use the connection from „Radio“ behind the Clovebox. There I can’t find so long the diagram. But I have attached you the manual to install the camera system originally. There are the correct instructions an a diagram to connect. I hope it’s helpful.
[EBA8201672189.PDF](https://github.com/cedricp/ddt4all/files/14418671/EBA8201672189.PDF)

**Comment by larskka** [comment]
Thank you so much :-) my fault i have trying to connect on the back of  r-link screen an not behandling the clovebox. So i learned a lesseon today

**Comment by AndyIreland** [comment]
so did you have to join your camera wires to the 32pin connector then?

**Reply by MouadSoufiani** [reply]
Hi? 

I need some schemes of the rlink2 (unit + screen) and the UCH. Can you help with it?

**Comment by larskka** [comment]
![20240227_211339](https://github.com/cedricp/ddt4all/assets/161332104/5b12086b-2217-45ae-9212-2b38e2388788)
Now i found a 32 pin over clovebox, connechted to a box in the right side.  But there is cable in the pins i think i should use Brown, yellow, red and black . Is it the right cable? If shall i just cut the cable and add the ones from camera? And finale number(page 13) 21 what is this plug?dosent look like the 32 pin. number 22(page 14) in the instruktion  what Are the cables A? Are you lookind in the plug from the behind where you plug in the cable?  Is 21 video + 22 video -  23 supply + 24 supply - ? Thank you for your time:-)

**Reply by zawavie2020** [reply]
i have the same issue brother , have you managed to find a solution brother ?

**Comment by Oussama3535** [comment]
Ineed pin location for clio5

**Reply by Furtif** [reply]
Is not ok this tutorial https://github.com/cedricp/ddt4all/discussions/1246 ?

**Comment by BlueRibbon23** [comment]
@matzel82  could you please help with the PIN/wiring diagram of the UCH for this VIN please? VF1RFB00163819901

Thank you.

**Comment by AndriyNakonechnyy** [comment]
Hello, could anybody help with renault espace 5 (2016 year) Telematic control unit connector pin diagram?

**Comment by Renotech1** [comment]
Why you need that for 😅

Op vr 1 aug 2025, 17:10 schreef AndriyNakonechnyy ***@***.***

**Reply by AndriyNakonechnyy** [reply]
This module is locked in my car and i don`t have live services and internet. So I want to try use another module but don`t want to remove cockpit for this if this is possible.

**Comment by Renotech1** [comment]
It hard to replace the tcu untit.. its register on renault servers ect
ect.. better buy new one or go to dealer.. if you want to chech wich
service are active an wich not you can contact me private then i look up
for you

Op vr 1 aug 2025, 17:47 schreef AndriyNakonechnyy ***@***.***

**Reply by AndriyNakonechnyy** [reply]
I have another one but i don`t now it status so i want to try connect it for checking status (i need tcu pinout for that reason). But if you can check my installed tcu it would be great. How can i contact you?

**Comment by Renotech1** [comment]
Hey thats not posible.  The tcu unit is linked to the car.. you can not use
it on an other car. Due not only vin. But 3 other numbers that are
registered on the renault servers..4g netwerk sw nr. Hw nr.... only a new
one can to this.. you can contact me private on me email for more
information

Op vr 1 aug 2025, 18:35 schreef AndriyNakonechnyy ***@***.***

**Reply by AndriyNakonechnyy** [reply]
I don't see your email 😔. It`s hidden. Maybe you can write me to andriy.nakon@gmail.com?

**Reply by AndriyNakonechnyy** [reply]
@Renotech1 hello, do you have any updates?

**Comment by JaZzpt** [comment]
Hello dear all, hope someone can help me. I have a Megane 4 from 2018 with rlink2 system. On the back of the radio I have a 24pin connector. The problem is that my camera came with a 32pin cable.... I've bought a 24pin cable, but it still don't work.. 
Does anyone know if like the scenic mentioned here,  I also have a 32 pin connector in other side, like behind the glovebox or something, to connect the camera? 

Kind regards

**Reply by BlueRibbon23** [reply]
Hello. Most probably you’ve connected to the display black connector . 

The camera connects to the 32pin connector of the Rlink 2 unit ( behind the screen there is a metal box, that is the Rlink 2 unit ) 

good luck
